react/components/SearchBar/components/AutocompleteInput.js

```diff
@@ -23,7 +25,10 @@ export default class AutocompleteInput extends Component {
   }
 
   componentDidMount() {
+    const { runtime } = this.props
     this.changeClassInput()
+    console.log('device', path(['hints', 'mobile'], runtime))
+    path(['hints', 'mobile'], runtime) && this.inputRef.current.focus()
```

Comment:
I do not think that this approach looks right. Bind this behavior every time on mobile mode, could compromise the component reusability in different scenarios and become a future issue. This should be configured via props, example: `<Search autoFocus={true} />`

Reply:
The focus on mobile should be something that always happens, it would be strange to have a props to decide if should focus or not.

Reply:
For example... Another component comes up and wants to reuse this search bar. In this new component, the search bar is always there... Think about it. This behavior would be undesirable.

Reply:
You made my mind, thinking this way it is a good choice to make this a choice by props